router/auth.go

```diff
@@ -167,9 +203,13 @@ func authInjectYellowfin(log *log.Logger, w http.ResponseWriter, req *http.Reque
 		done := false
 		target.lock.RLock()
 		token_yf, exists := target.tokenMap[authData.Identity].(*yellowfinToken)
-		if exists && token_yf.Expires.After(time.Now()) {
-			done = true
-			inject(token_yf)
+		yfLoginparams := yellowfinLoginParameters{}
+		if exists {
+			yfLoginparams = token_yf.Parameters
```

Comment:
Should be a comment here mentioning that even if the session has expired, we still want to retain the login parameters.